### DISCUSSION TITLE
How to add images (PNG/JPG) in the axis labels?

### DISCUSSION THREAD

**Discussion by ggregoire** [plouc/nivo#2564] [Q&A]
Hi,

I was wondering if there is a way to add images (PNG/JPG) in the axis labels?

I have several charts made with Echarts for React and I'm trying to migrate these charts to Nivo.

For example:

![Screenshot 2024-04-15 at 10 04 42](https://github.com/plouc/nivo/assets/591804/cdb93dbe-b1f3-4b48-a58b-bbe8adac7c0d)

In Echarts it's done by setting the background image of the labels.

**Comment by plouc** [comment]
You can use the renderTick attribute for the corresponding axis config.
